app-frontend/src/app/components/projects/annotateSidebarItem/annotateSidebarItem.html

```diff
@@ -50,21 +50,24 @@
 <div class="list-group-item" ng-if="!$ctrl.annotation.geometry || ($ctrl.editId === $ctrl.annotation.properties.id)">
   <div class='annotation-new'>
     <form>
-    <angucomplete-alt pause="100"
-                      initial-value="$ctrl.annotation.properties.label"
-                      selected-object="$ctrl.labelObj"
-                      placeholder="{{$ctrl.annotation.properties.label || 'Label name...'}}"
-                      local-data="$ctrl.labelInputs"
-                      search-fields="name"
-                      title-field="name"
-                      minlength="1"
-                      input-class="form-control annotation-label"
-                      match-class="annotation-highlight"
-                      override-suggestions="true"
-                      ng-class="{'is-invalid': $ctrl.isInvalid }">
-    </angucomplete-alt>
-    <textarea class="form-control"
+      <input type="text"
+             class="form-control label-name-input"
+             ng-init="$ctrl.labelNameInput = $ctrl.annotation.properties.label"
+             ng-model="$ctrl.labelNameInput"
+             ng-change="$ctrl.onLabelNameChange()"
+             ng-class="{'is-invalid': $ctrl.isInvalid }"
+             placeholder="{{$ctrl.annotation.properties.label || 'Label name...'}}">
+      <div class="label-name-selections"
+           ng-if="$ctrl.showMatchedLabels">
+        <div class="label-name-option"
```

Comment:
We'll probably want to limit the number of potential matches (3 or 4) and sort them shortest to longest.